app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] As you have done in the post the foreign key is author_id, so you join it, Please 🙏🏽  correct this line to join the  `like` table to the correct column 👇🏽 

```ruby
belongs_to :author, class_name: 'User'
```

Reply:
Hi @franclobo ,
Thanks for your kind, humble, and detailed review. I appreciate the efforts you have tried putting in to explain your suggestions. However, I request you to kindly think about your suggestions as I think the changes you suggested are not mentioned in the project requirements. If I am mistaken, kindly explain to me further as to how am I mistaken. Along with this comment, I am writing two other comments below so please check them too.

As mentioned in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), the `author` is used only for `Post`. The `Like` has the attribute which goes by the name `UserID`.

Reply:
Hello, @sameedmohsin28.

You have the reason in your point, as CR I have been reviewing according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). I hope the people in charge can change to use the same ERD diagram. For this reason I have changed the required change to OPTIONAL.

Thanks for your suggestion.